packages/next-mdx/src/get-nodes.ts

```diff
@@ -181,6 +182,11 @@ export async function getFileData(file: MdxFile): Promise<MdxFileData> {
 
   const { content, data: frontMatter } = matter(raw)
 
+  const word_count = content.split(/\s+/gu).length
+  const reading_time = readingTime(content)
+  frontMatter.reading_time = reading_time
```

Comment:
I think we should keep `frontMatter` as is and add `readingTime` to the `Node` interface.

https://github.com/arshad/next-mdx/blob/b2d3c5e57f1fffea3b05e0832a43fdb99f69000d/packages/next-mdx/src/get-nodes.ts#L20-L23

See also https://github.com/ngryman/reading-time/blob/master/index.d.ts for the types.